DAQ_System/Source/Application/DataLogger/SdDataLogger.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+* SD Data Logger
+* Authors:        Cristian Cruz
+*                 Ethan Fischer
+*
+* Email:          cris14.alex@gmail.com
+*                 ejf180001@utdallas.edu
+* 
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#include "SdDataLogger.hpp"
+#include "FATFileSystem.h"
+#include "SDBlockDevice.h"
+
+namespace application {
+
+// block device and file system declarations
+SDBlockDevice block_device_;
+FATFileSystem file_system_;
+FILE* data_file_;
```

Comment:
Can we remove these global variables?

Reply:
Done, I didn't even realize we didn't need those. It's always been my understanding that despite something being in the header file, you still have to initialize it in the source file so that it knows it exists basically but commenting those out doesn't produce any errors so we're good there.

Reply:
This was handled by the initializer list in the constructor